File: CDTextViewPlaceHolderDemo/CDTextViewPlaceHolderDemo/ViewController.m
//
//  ViewController.m
//  CDTextViewPlaceHolderDemo
//
//  Created by chdo on 2017/6/6.
//  Copyright © 2017年 chdo. All rights reserved.
//

#import "ViewController.h"
#import "UITextView+CDPlaceHolder.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    
    self.view.backgroundColor = [UIColor cyanColor];
    
    UITextView * te = [[UITextView alloc] initWithFrame:CGRectMake(50, 50, 200, 200)];
    
    [te setUpPlaceHolder:@"aaaa" font:nil textColor:nil];
    
    [self.view addSubview:te];
    
    
    
}


- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}


@end

File: CDTextViewPlaceHolderDemo_swift/CDTextViewPlaceHolderDemo_swift/ViewController.swift
//
//  ViewController.swift
//  CDTextViewPlaceHolderDemo_swift
//
//  Created by chdo on 2017/6/7.
//  Copyright © 2017年 chdo. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        let textv = UITextView(frame: CGRect(x: 50, y: 50, width: 200, height: 200))
        view.backgroundColor = UIColor.cyan
        view.addSubview(textv)
        
        
        textv.setUpPlaceHoldr("aaa")
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

